packages/sources/finnhub-secondary/test/integration/adapter-ws.test.ts

```diff
@@ -21,6 +21,13 @@ const mockWebSocketServer = (url: string) => {
               t: 1641035471111,
               v: 0,
             },
+            {
```

Comment:
tried looking at the finnhub docs but they were a little unclear, so 2 questions:

1. is this mock returning 2 separate messages or 2 objects that are part of the same response data in a single message?
2. (follow up) is that the expected or observed behaviour from the Finnhub WS API?

Reply:
This is a single WebSocket message, which can contain multiple trades for different assets in the `data` array - this matches what Finnhub WS returns if subscribed to multiple symbols.